spec/controllers/api/v1/users/checkins_controller_spec.rb

```diff
@@ -120,24 +121,25 @@
 
       it "fetches the user's last device checkin with all attributes" do
         get :last, params: params
-        expect(res_hash.first['id']).to be checkin.id
-        expect(res_hash.first.keys).to match checkin.attributes.keys
+        expect(res_hash[:checkins].first['id']).to be checkin.id
+        # TODO: define required set of attributes
+        # expect(res_hash[:checkins].first.keys).to match checkin.attributes.keys
```

Comment:
The check is because this test is for when requesting checkins with the secret copo-app-key present which [bypasses sanitizing](https://github.com/earlymarket/Coposition/blob/32dcc2bb577c7b2fa99b293cf70502177ba114be/app/models/device.rb#L37). This is so Coposition can use this endpoint but get unsanitized info.